zanata-client-commands/src/main/resources/prompts.properties

```diff
@@ -89,3 +89,5 @@ no.default.mapping=Can not find default translation file mapping rule for projec
 invalid.rule=Rule defined is not valid (must contain locale): %s
 unrecognized.variables=Valid variables are %s. Anything else in '%s' will be treated as literal value.
 confirm.rule=Continue with current mapping rule(s) (y/n)?
+
+locales.in.config.deprecated=Locale mappings in project config file (zanata.xml) is deprecated! Please remove it from the file so that it can use latest language settins on the server.
```

Comment:
:speech_balloon: "Locale mappings in project config file (zanata.xml) are deprecated! Please set locale mappings in the server, then remove the <locales> section from the config file."

We could provide a link to the project/version language settings page where they would enter the settings.